recipe/migrations/r410.yaml

```diff
@@ -0,0 +1,16 @@
+migrator_ts: 1621536224
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  automerge: true
+
```

Comment:
Add the long-term key for this one. See the other long running migrators like aarch

Reply:
`longterm: True`